packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx

```diff
@@ -252,7 +252,7 @@ export const menuTabs: ForumOptions<Array<MenuTab>> = {
       showOnCompressed: true,
     }, {
       id: taggingNamePluralSetting.get(),
-      title: taggingNamePluralCapitalSetting.get() + " Wiki",
```

Comment:
Also change "Topics Wiki" to "Topics", which arguably doesn't make sense in this PR but we were going to do it anyway